.ci/jenkins/project/Jenkinsfile.nightly

```diff
@@ -49,6 +53,21 @@ pipeline {
             }
         }
 
+        // Applies only to the 9.x branch build. The 9.?.x release branches must remain intact.
+        stage('Migrate to 9') {
+            when {
+                expression { return is9xBranch() }
+            }
+            steps {
+                script {
+                    checkoutRepo(optaPlannerRepository, optaPlannerMainBranch)
+                    checkoutNewBranch(optaPlannerRepository, optaPlannerMainBranch, optaPlanner9xBranch)
+                    callMigrationScript(optaPlannerRepository)
+                    forcePushBranch(optaPlannerRepository, optaPlanner9xBranch)
+                }
+            }
+        }
+
```

Comment:
In the next stage `quickstartsBranch` calculation should be changed to adapt to 9.x

Reply:
thanks; I think we should map both `main` and '9.x' to `development`; later, in the deploy/promote jobs, the stages will figure out what to do.